### PR TITLE
References refactoring

### PR DESCRIPTION
**Pull request by alan-null**
- Removed unused references
- Excluded references from publish
- Removed dll version constraints

### CODE REVIEW

File: src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj [hunk 1]
```diff
@@ -54,14 +54,6 @@
...
       <Private>False</Private>
     </Reference>
-    <Reference Include="HtmlSanitizationLibrary, Version=4.3.0.0, Culture=neutral, PublicKeyToken=d127efab8a9c114f, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\AntiXSS.4.3.0\lib\net40\HtmlSanitizationLibrary.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Lucene.Net">
-      <HintPath>$(SitecorePath)\bin\Lucene.Net.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj]: We need this reference for HTML encoding of user comments

File: src/Sitecore.Modules.WeBlog.WebForms/Sitecore.Modules.WeBlog.WebForms.csproj [hunk 2]
```diff
@@ -141,13 +125,15 @@
...
     <Reference Include="System.Web" />
     <Reference Include="System.Web.Extensions" />
-    <Reference Include="System.Web.Http, Version=5.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Web.Http">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(SitecorePath)\bin\System.Web.Http.dll</HintPath>
+      <Private>False</Private>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=4.0.0.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Web.Mvc">
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog.WebForms/Sitecore.Modules.WeBlog.WebForms.csproj]: We shouldn't need MVC in WebForms should we?

File: src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj [hunk 3]
```diff
@@ -141,13 +130,15 @@
...
     <Reference Include="System.Web" />
     <Reference Include="System.Web.Extensions" />
-    <Reference Include="System.Web.Http, Version=5.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Web.Http">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(SitecorePath)\bin\System.Web.Http.dll</HintPath>
+      <Private>False</Private>
     </Reference>
-    <Reference Include="System.Web.Mvc, Version=4.0.0.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+    <Reference Include="System.Web.Mvc">
```
Comment (adeneys) [inline src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj]: We shouldn't need MVC reference here as all MVC stuff should be contained to the MVC project.

File: src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj (hunk 1 above)
Comment (alan-null) [inline src/Sitecore.Modules.WeBlog/Sitecore.Modules.WeBlog.csproj]: Are you talking about `HtmlSanitizationLibrary`? 
It is delivered with `AntiXssLibrary.xml` but is not used in a part of API used by us.
I just created build env without this DLL and I am able to display comments without any problems.

File: src/Sitecore.Modules.WeBlog.WebForms/Sitecore.Modules.WeBlog.WebForms.csproj (hunk 2 above)
Comment (alan-null) [inline src/Sitecore.Modules.WeBlog.WebForms/Sitecore.Modules.WeBlog.WebForms.csproj]: Done

**Pull request merged**